guides/v2.3/graphql/functional-testing.md

```diff
@@ -196,6 +196,50 @@ $registry->unregister('isSecureArea');
 $registry->register('isSecureArea', false);
 ```
 
+### Fixture configs
+
+Use `@magentoConfigFixture` annotation to set a custom config value. It supports a `store` scope only.
+ 
+#### Syntax
+
+```php
+/**
+ * @magentoConfigFixture <store_code>_store <config_key> <config_value>
+ */
+```
+
+where
+- `<store_code>` - Code of store. See `store`.`code` table
```

Comment:
I would make the description more transparent here in the following way for all arguments
```suggestion
- `<store_code>` - Store code. See the `store`.`code` database field value. 
```

Reply:
**fixed**

Reply:
@rogyar , thank you for review!

All your suggestions have been implemented.

Thank you!